motorSupportApp/src/devMotorAxis.cpp

```diff
@@ -430,17 +427,13 @@ asynStatus devMotorAxis::poll(bool *moving) {
 	// Get the actual position
 	scaleValueToMotorRecord(&st_axis_status.fActPosition);
 	setDoubleParam(pC_->motorPosition_, st_axis_status.fActPosition);
-	setDoubleParam(pC_->motorEncoderPosition_, st_axis_status.fActPosition);
 
-	// Calculate if moving and set appropriate bits
-	nowMoving = st_axis_status.bMoving;
-	setIntegerParam(pC_->motorStatusMoving_, nowMoving);
-	setIntegerParam(pC_->motorStatusDone_, !nowMoving);
-	*moving = nowMoving ? true : false;
+  // Calculate if moving and set appropriate bits
+  int nowMoving = st_axis_status.bMoving;
+  setIntegerParam(pC_->motorStatusMoving_, nowMoving);
+  setIntegerParam(pC_->motorStatusDone_, !nowMoving);
+  *moving = st_axis_status.bMoving ? true : false;
```

Comment:
I don't understand why this has changed - it doesn't look like `st_axis_status` can change between the `int nowMoving = ...` declaration and this line? Why are we unable to use `nowMoving` as before?